sdk/tables/data-tables/review/data-tables.api.md

```diff
@@ -318,6 +350,20 @@ export class TableClient {
     readonly tableName: string;
     updateEntity<T extends object>(entity: TableEntity<T>, mode: UpdateMode, options?: UpdateTableEntityOptions): Promise<UpdateEntityResponse>;
     upsertEntity<T extends object>(entity: TableEntity<T>, mode: UpdateMode, options?: UpsertTableEntityOptions): Promise<UpsertEntityResponse>;
+    }
+
+// @public
+export interface TableClientLike {
```

Comment:
Just wondering - do we need to export this as part of our public surface area?  It didn't seem to be referenced anywhere else in this review.md

Reply:
Yeah I don't think we actually need to export this type.

Reply:
You're right I just needed this to avoid a circular dependency in TableBatch, moving this to internalModels